lib/Doctrine/ORM/Mapping/EntityListenerResolver.php

```diff
@@ -25,12 +23,12 @@ public function clear($className = null);
      *
      * @return object An entity listener
      */
-    public function resolve($className);
+    public function resolve(string $className): object;
 
     /**
      * Register a entity listener instance.
      *
      * @param object $object An entity listener
      */
-    public function register($object);
+    public function register(object $object): void;
```

Comment:
Do you want to keep the arguments `@param` docblock?

Reply:
Not necessarily. I was unsure if it still provides value. What do you think? Remove?

Reply:
Yes, it can be removed.

Reply:
Done.